IfIsNotInA.java

```diff
@@ -0,0 +1,35 @@
+
+
+
+
+            /*21.	Given two array lists A and B, A has an element that is not in B while the other
+             elements are the same in both two lists. Find that missing element in B.
+              (This is a live coding challenge. Any programming language is fine.)*/
+
+
+import java.util.*;
+
+public class IfIsNotInA {
+    public static List solution(int[] first, int[] second) {
+        ArrayList<Integer> firstArrayList = new ArrayList<Integer>();
```

Comment:
Avoid using "Array<Integer>" if possible when solving algorithm questions, because it's an array of Boxed type and it could degrade the performance of your solution.

https://dzone.com/articles/java-performance-notes-autoboxing-unboxing

https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/Performance-cost-of-Java-autoboxing-and-unboxing-of-primitive-types